larreco/WireCell/CellTree_module.cc

```diff
@@ -71,6 +71,7 @@ namespace wc {
     void processOpFlash(const art::Event& evt);
     void processSpacePoint(const art::Event& event, TString option, ostream& out = cout);
     void processSpacePointTruthDepo(const art::Event& event, TString option, ostream& out = cout);
+    void processSpacePointTruthDepoT(const art::Event& event, TString option, ostream& out = cout);
```

Comment:
Is it possible to adjust the existing `processSpacePointTruthDepo` function so that you if a user specifies `saveJSON_T: true`, the function does the right thing?  Right now, both functions `processSpacePointTruthDepo(T)` have a lot of the same code, and I'm just wondering if there is a way to reduce the duplicated code.

Reply:
Thanks Kyle. I agree with you. I reduced the duplicated code and added a parameter to the original function to fulfill my goal.